taichi/python/export_lang.cpp

```diff
@@ -1106,7 +1107,14 @@ void export_lang(py::module &m) {
            py::return_value_policy::reference)
       .def("get_quant_float_type", &TypeFactory::get_quant_float_type,
            py::arg("digits_type"), py::arg("exponent_type"),
-           py::arg("compute_type"), py::return_value_policy::reference);
+           py::arg("compute_type"), py::return_value_policy::reference)
+      .def(
+          "get_tensor_type",
+          [&](TypeFactory *factory, std::vector<int> shape,
+              const DataType &element_type) {
+            return factory->create_tensor_type(shape, element_type);
```

Comment:
Why not use `TypeFactory::get_tensor_type`?

Reply:
Oh it's for convenient use at python-scope.

Basically, `TypeFactory::get_tensor_type` takes "Type*" and returns "Type*". However in python-scope, we usually use "DataType", and you have to write sth like:

```
if isinstance(type, _ti_python_core.DataType):
        type = type.get_ptr()
type = _type_factory.get_tensor_type(bits, signed, type)
return cook_type(type)
```

Reply:
Got it.